http2/src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2SampleResult.java

```diff
@@ -181,13 +200,94 @@ public void notifySample() {
         }
       }
       if (!sonsArePendingResponse) {
+        runPostProcessors(postProcessors);
+        checkAssertions(assertions);
         SampleEvent event = new SampleEvent(this, getThreadName(),
             threadVars, false);
-        listenerNotifier.notifyListeners(event, pack.getSampleListeners());
+        listenerNotifier.notifyListeners(event, this.sampleListeners);
       }
     }
   }
 
+  private void runPostProcessors(List<PostProcessor> extractors) {

```

Comment:
name extractors is a little confusing since there might be post processors that aren't extractor, right?